website/style_guide.md

```diff
@@ -262,3 +262,39 @@ the first column of the comment. For example:
 
 Code examples should not contain additional comments. It is already inside a
 comment. If it needs further comments is not a good example.
+
+## Each module should come with tests
+
+Each module should come with its test as a sibling with the name
+`modulename_test.ts`. For example the module `foo.ts` should come with its
+sibling `foo_test.ts`.
+
+## Unit Tests should be explicit
+
+For a better understanding of the tests, function should be correctly named as
+its prompted throughout the test command. Like:
+
+```
+test myTestFunction ... ok
+```
+
+Example of test:
+
+```ts
+import { assertStrContains } from "https://deno.land/std@v0.3.1/testing/asserts.ts";
+import { test } from "https://deno.land/std@v0.3.1/testing/mod.ts";
+
+test(function myTestFunction() {
+  assertStrContains("Denosaurus", "saur");
+  assertStrContains("Denosaurus", "Deno");
+  assertStrContains("Denosaurus", "rus");
+  let didThrow;
+  try {
+    assertStrContains("Denosaurus", "Raptor");
+    didThrow = false;
+  } catch (e) {
+    didThrow = true;
+  }
+  assertEquals(didThrow, true);
```

Comment:
Please use a smaller one line example with just a single `assertEquals`... I think all this code distracts from the point the style guide is making. This isn't documentation.

Reply:
edited